### ISSUE TITLE
Course Module: Misc issues

### ISSUE THREAD

**Issue by kstsvcs** [kstsvcs/ics#1286]
on page, course/gridlist
- Facilator List CRUD dialog should be called Add Facilator instead of Instructor and the category should show all valid values (including those from AC)

on App,
- Assuming student submits the (subjective) answersheet offline then the evaluator should be able to upload a scanned corrected copy of the same and the student should be see it (with the given workflow as below)
- Student's submission of subjective assessment is not visible to the evaluator
- Evalator/student should see AssessmentCode as one of the main columns
- Attendance coordinator should be able to take attendance (As per one of the events of the course) from the app (with offline capability)
- Overall Attandance score calculation: At the corse level, the admin should be able to generate attendance summary. It would then trigger calculation of finding out (category wise, ignoring the ones for assessment) : total classes and num attended per student...the same would then be store as scrore in the corresponding IA
- Overall Report download should also do total (per category) and overall total .

workflow for IA
- the Meaning of varios IA statsu is as under:
- "PRE-REGISTRATION" -> student's entry is created. As soon as this is done ie a new IA is created, the IA.assessmentCode should be generated (unique 6 char alphanumeric)
- ,"REGISTERED" -> all eligible students would be marked as such and only these would be eligible to take test / see ans sheet
- ,"READY" -> only IAs of this status would be visible to Evaluator for downloading/ uploading scanned answer sheet
- , "STARTED" -> Evaluator would marks 'Started' when s/he has started the evalation
- , "SUBMITTED" -> Evaluator would marks 'Submitted' when s/he has commpleted the evalation and uploaded the ans sheet
- , "CANCELLED"- > IA can be only cancelled by admin (and only admin can put comments)
- "PARKED" -> IA is parked once Admin is ready to publish the result..now students can see their corrected ans sheet and marks
- , "RESUMED" -> For future use (maybe re-correction scenario)
